tests/conftest.py

```diff
@@ -1848,6 +1856,7 @@ def update_dvs(log_path, new_dvs_env=[]):
             dvs.destroy_servers()
             dvs.create_servers()
             dvs.restart()
+            time.sleep(60)
```

Comment:
Why do we need sleep here?

Reply:
I added a delay here to ensure that once the DVS switch starts, there is a pause before it performs any further actions, allowing sufficient time for the ports to initialize properly.